conf/entry_location_rocksdb.conf.default

```diff
@@ -26,7 +30,7 @@
  # set by jni: options.setLogFileTimeToRoll
  log_file_time_to_roll=86400
  # set by jni: options.setMaxBackgroundJobs or options.setIncreaseParallelism
- max_background_jobs=2
+ max_background_jobs=32
```

Comment:
What's the rationale in making this change in this PR?

Reply:
> What's the rationale in making this change in this PR?

Hi Lari:
The previous default value for **max_background_jobs** was written incorrectly and was not consistent with the default value in the code, so this PR corrected it. Refer to: https://github.com/apache/bookkeeper/blob/999cd0f2ab14404be4d6c24e388456dbe56bb1a8/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java#L213

Reply:
Thanks for explaining @AuroraTwinkle